crates/release_plz_core/src/gitea_client.rs

```diff
@@ -79,6 +85,31 @@ impl<'a> GiteaClient<'a> {
         Ok(())
     }
 
+    pub async fn create_gitea_release(
+        &self,
+        git_tag: String,
+        release_body: &str,
```

Comment:
from the [API](https://try.gitea.io/api/swagger#/repository/repoCreateRelease) I see this fiels is called `body`, not `release_body`.
Was this a typo or maybe there's a reason behind this name? 🤔 
![image](https://user-images.githubusercontent.com/11428655/211922370-a4c0a0ae-0750-4436-bced-1323f95acd84.png)